MANIFEST.in

```diff
@@ -5,6 +5,8 @@ recursive-include vpi_shim *.c Makefile
 recursive-include examples *.v *.py Makefile *.tcl *.cfg
 recursive-include scripts *
 recursive-include simulator *.c *.h Makefile
+recursive-include lib *.c* *.h Makefile
+recursive-include include *.h
 recursive-include modules *.py
 recursive-include makefiles Makefile.*
-include Makefile README.md
+include Makefile README.md version
```

Comment:
The `MANIFEST.in` file only needs to list files which appear in the source package (sdist), but not in the binary package (bdist). That is, all files which are already included in some way in the setup.py file, e.g. in  `package_data` should not be added here.

This leaves most likely only the examples.

Reply:
I tried to keep it minimal that seems to work.